setup.py

```diff
@@ -17,14 +22,14 @@ def get_version():
     """
     locals_dict = {}
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
+    with open(os.path.join(__curdir__, 'nilearn', 'version.py')) as fp:
```

Comment:
I'm not sure at all, beyond testing--I'm not so familiar with `setup.py`.  Semantically, for me, this is simply cleaner and safer--those files are in a specific location, not relative to the current working directory.